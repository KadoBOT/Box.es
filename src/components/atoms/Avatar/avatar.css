.avatar {
  height: 60px;
  width: 60px;
  min-width: 60px;
  background-size: cover !important;
  border-radius: 5px;
  background-position: center !important;
}
