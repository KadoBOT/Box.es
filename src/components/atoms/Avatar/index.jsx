import React, {PropTypes} from 'react'

import './avatar.css'

const Avatar = ({img}) => <div className="avatar" style={{background: `url(${img})`}} />

Avatar.propTypes = {
  img: PropTypes.string.isRequired
}

export default Avatar
