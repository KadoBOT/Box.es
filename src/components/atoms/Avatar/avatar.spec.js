import React from 'react'
import renderer from 'react-test-renderer';
import Avatar from './index'

it('should render the Avatar', () => {
    const wrapper = renderer.create(
        <Avatar img="colossus.jpg" />
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
});
