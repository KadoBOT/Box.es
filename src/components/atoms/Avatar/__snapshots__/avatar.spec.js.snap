exports[`test should render the Avatar 1`] = `
<div
  className="avatar"
  style={
    Object {
      "background": "url(colossus.jpg)",
    }
  } />
`;
